src/customTypes/id.js

```diff
@@ -0,0 +1,7 @@
+const { field } = require('../field')
+
+const id = (type, options) => {
+  return field(type, { ...options, isId: true })
```

Comment:
```js
const Plan =
    entity('Plan', {
        ...
        myId: id(Number, { validation: { length: { is: 10 }  }),
```
Is it possible to have the above code with this implementation?

Reply:
Yes, should work fine. I've added tests with this scenario